README.md

```diff
@@ -41,7 +41,7 @@ details please see the [LICENSE](./LICENSE) file or http://juliusbaxter.net/ohdl
 
 A CPU core cannot be trusted without a full set of verification testing.  The
 `or1k_marocchino` is constantly verified for correctness with the or1k Continuous
-Integration (CI) suite running in [travis ci](travis-ci.org).  This currently covers:
```

Comment:
Why remove this link?

Reply:
Because it is redirected to this link: https://github.com/openrisc/or1k_marocchino/blob/master/travis-ci.org

Reply:
I see. Thanks for fixing